docs/src/content/docs/resources/plugins.mdx

```diff
@@ -56,7 +56,7 @@ Extend your site with official plugins supported by the Starlight team and commu
 	<LinkCard
 		href="https://astro-ghostcms.xyz/intro/starlight/install/"
 		title="starlight-ghostcms"
-		description="Add your GhostCMS blog posts alongside your Starlight Docs"
+		description="Add your GhostCMS blog posts alongside your Starlight docs."
 	/>
```

Comment:
We can do this in a follow-up PR but this URL is a 404 and the plugin is deprecated so we should probably remove this.